api/server.ts

```diff
@@ -39,20 +39,15 @@ app.use(bodyParser.json());
 app.use(passport.initialize());
 
 passport.use(new BasicStrategy(async (username, password, done) => {
-    var sql = "SELECT client_id, api_key " +
-              "FROM SecurityTokens " +
-              "WHERE client_id = @clientId and api_key = @apiKey";
+    
     try {
-        var results = await tds.default.sql(sql)
-            .parameter('clientId', TYPES.NVarChar, username)
-            .parameter('apiKey', TYPES.NVarChar, password)
-            .executeImmediate();
-        if (!results || results.length != 1) {
-            return done(null, false, {message: 'Invalid client_id or api_key'});
-        }
-        else if (username.valueOf() == results[0].client_id && password.valueOf() == results[0].api_key) {
-            return done(null, true);
-        }
+        var connStrings = process.env.Basic_Auth_Conn_String;
+        var basicAuthCreds = JSON.parse(connStrings);
```

Comment:
Consider parsing the JSON from the env vars once & saving the result in a variable. This will be much more efficient than parsing the JSON (which is effectively const) for every request.